libs/mark-flow-ui/src/components/election_info.tsx

```diff
@@ -66,7 +66,7 @@ export function ElectionInfo({
 
   return (
     <Container>
-      <Seal seal={seal} />
+      <Seal seal={seal} maxWidth="7rem" />
```

Comment:
Previously this seal was a constant 250px across size settings. I experimented and found 7rem to be the most pleasing.

### Before
<img width="1155" alt="Screen Shot 2023-11-21 at 5 42 57 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/78196410-2548-49c8-8f5c-47989d5801a5">
<img width="1153" alt="Screen Shot 2023-11-21 at 5 42 44 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/6b431114-caa9-43de-94b4-81a009069f43">
<img width="1153" alt="Screen Shot 2023-11-21 at 5 42 29 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/65621db3-5292-434b-8aa8-b9cfc6650da1">
<img width="1155" alt="Screen Shot 2023-11-21 at 5 42 18 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/76b8aeae-6976-4e76-9a00-e830bbbbae67">

### After
<img width="1157" alt="Screen Shot 2023-11-21 at 5 41 32 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/0819be4f-a88c-4a8a-acaf-b215ed0a8361">
<img width="1152" alt="Screen Shot 2023-11-21 at 5 41 21 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/024485d2-b17e-4cf7-a10c-2b9d84b43ea5">
<img width="1154" alt="Screen Shot 2023-11-21 at 5 41 12 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/43420466-c71e-4859-b994-47d2d33cbaf9">
<img width="1153" alt="Screen Shot 2023-11-21 at 5 41 01 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/5897f65a-f242-4f4f-b05f-d1c1428584a3">

Reply:
After adding some padding in both orientations:

<img width="380" alt="Screen Shot 2023-11-22 at 2 18 38 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/02d9177e-189f-48e0-8502-a6b972433a42">
<img width="379" alt="Screen Shot 2023-11-22 at 2 18 28 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/8a5b0d71-96db-48df-848d-357b52645d64">
<img width="378" alt="Screen Shot 2023-11-22 at 2 18 14 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/ee7b6a4b-e76c-4646-a0cb-981c89773c1a">
<img width="376" alt="Screen Shot 2023-11-22 at 2 18 00 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/2b32b22b-b3b7-42fd-a7b0-1c16575dc1c4">
<img width="1153" alt="Screen Shot 2023-11-22 at 2 13 39 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/7f801d45-6025-4d62-8565-2245c6f9630f">
<img width="1154" alt="Screen Shot 2023-11-22 at 2 13 30 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/09c72e21-256e-43b2-a4fe-6638ec2dcd9d">
<img width="1155" alt="Screen Shot 2023-11-22 at 2 13 20 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/b00d5bf8-8c26-437a-91a2-db53e6dc9d97">
<img width="1154" alt="Screen Shot 2023-11-22 at 2 13 11 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/dfed8fce-4974-44d0-be35-e01a5214ffd7">